packages/lodestar/src/chain/forkChoice/forkChoice.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @module chain/forkChoice
+ */
+
+import {toHexString} from "@chainsafe/ssz";
+import {BeaconState, Slot} from "@chainsafe/lodestar-types";
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {ForkChoice, ProtoArray} from "@chainsafe/lodestar-fork-choice";
+
+import {computeAnchorCheckpoint} from "../initState";
+import {ChainEventEmitter} from "../emitter";
+import {ForkChoiceStore} from "./store";
+
+/**
+ * Fork Choice extended with a ChainEventEmitter
+ */
+export class LodestarForkChoice extends ForkChoice {
+  constructor({
+    config,
+    emitter,
+    currentSlot,
+    anchorState,
+  }: {
+    config: IBeaconConfig;
+    emitter: ChainEventEmitter;
+    currentSlot: Slot;
+    anchorState: BeaconState;
+  }) {
+    const {blockHeader, checkpoint} = computeAnchorCheckpoint(config, anchorState);
+    const justifiedCheckpoint = {...checkpoint};
+    const finalizedCheckpoint = {
+      ...checkpoint,
+      // If not genesis epoch, justified checkpoint epoch must be set to finalized checkpoint epoch + 1
```

Comment:
the code does not go with the comment, maybe switch `finalizedCheckpoint` and `justifiedCheckpoint` assignment?

Reply:
created https://github.com/ChainSafe/lodestar/pull/1735/files